common/src/main/java/com/microsoft/identity/common/internal/cache/ICacheKeyValueDelegate.java

```diff
@@ -27,15 +27,46 @@
 import com.microsoft.identity.common.internal.dto.Credential;
 
 public interface ICacheKeyValueDelegate {
-
+    /**
+     * Generate cache key for a specific account.
+     *
+     * @param account Account
+     * @return String
+     */
     String generateCacheKey(final Account account);
 
+    /**
+     * Generate cache value for a specific account.
+     *
+     * @param account Account
+     * @return String
+     */
     String generateCacheValue(final Account account);
 
+    /**
+     * Generate cache key from the credential.
+     *
+     * @param credential Credential
+     * @return String
+     */
     String generateCacheKey(final Credential credential);
 
+    /**
+     * Generate cache value from the credential.
+     *
+     * @param credential Credential
+     * @return String
+     */
     String generateCacheValue(final Credential credential);
 
+    /**
+     * Get the account credential from cache value.
+     *
+     * @param string String
+     * @param t      AccountCredentialBase
```

Comment:
extra tab

Reply:
This is not a tab. This is auto alignment for the Javadoc.

Thank you for the CR @addev-ashish, I have addressed the rest as well as all other occurrences in the code.